scripts/forc-check.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 main_dir=$(pwd)
-forc_tomls=$(find . -type f -name "Forc.toml")
+forc_tomls=$(find . -type f -name "Forc.toml  -path './packages/forc/sway-repo' -prune")
```

Comment:
Could you explain this change? Adding `echo $forc_tomls` with this change present gives back no `Forc.toml`s to check in the script.

Reply:
Good catch! b2e45e5f19282b9b83b342c7274c89f92e052398

Reply:
It might be good to somehow verify that the scripts are actually running on a list of `Forc.toml`s and aren't passing the CI just because no `Forc.toml`s were found.